lib/pacemaker/pcmk_sched_clone.c

```diff
@@ -245,14 +245,17 @@ can_interleave(const pcmk__colocation_t *colocation)
  * \param[in]     primary        Primary resource in colocation
  * \param[in]     colocation     Colocation constraint to apply
  * \param[in]     for_dependent  true if called on behalf of dependent
+ *
+ * \return 0
```

Comment:
?

Reply:
Thanks, I thought I got all of these. In a first draft, I was returning nonzero only from the primitive method. I thought I only needed to check it within the group-with method.